Essentials/src/messages_en.properties

```diff
@@ -353,6 +353,7 @@ noMetaJson=JSON Metadata is not supported in this version of Bukkit.
 noMetaPerm=§4You do not have permission to apply §c{0}§4 meta to this item.
 none=none
 noNewMail=§6You have no new mail.
+noPayBlocked=§6You are not blocking anyone''s pay.
```

Comment:
Messages should be in messages.properties only, as the localized versions will be overwritten by Crowdin.